backend/user_management/routers/admin.py

```
@@ -0,0 +1,153 @@
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Product not found")
+
+    db.delete(product)
+    db.commit()
+    return {"detail": "Product deleted successfully"}
+
+#Add order management functionality for admin
+class OrderUpdate(BaseModel):
+    status: str
+
+@router.get("/orders")
+async def get_all_orders(db: Session = Depends(get_db), current_admin: User = Depends(get_current_admin)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_all_orders`

(ANN201)